ontology/yaml/resources/SAFETY/entity_types/GENERALTYPES.yaml

```diff
@@ -75,4 +75,9 @@ LDS:
   is_abstract: true
   implements:
   - EQUIPMENT
-
+FSS:
```

Comment:
probably needs to be added to [BDNS](https://github.com/theodi/BDNS/blob/master/BDNS_Abbreviations_Register.csv)

Reply:
Ohkay Thank you

Reply:
Incorporated in BDNS